### PR TITLE
comments in english

### PR DESCRIPTION
**Pull request by anjok**

### CODE REVIEW

**Comment by anjok** [comment]
Note: I did nothing more than a bunch of pasting into translate.google.com, copying out and pasting it into the python files. Then I installed the requirements and ran "python3 -m rcute_cozmars" and fixed things until it ran. I didn't read the translations in detail, but they look ok to me. At least they make more sense than Chinese characters :)

**Comment by anjok** [comment]
I also have a version of the docs in English as a PDF, you can paste the URL into translate.google.com like this:

https://translate.google.com/translate?hl=&sl=zh-CN&tl=en&u=https%3A%2F%2Frcute-cozmars.readthedocs.io%2F_%2Fdownloads%2Fzh_CN%2Flatest%2Fpdf%2F

and get it in one book. Looks neat :)

**Comment by anjok** [comment]
You might want to write that into your README.md

File: rcute_cozmars/button.py [hunk 1]
```diff
@@ -1,57 +1,56 @@
-from . import util
```
Comment (hyansuper) [inline rcute_cozmars/button.py:1]: Don't change any code，they are tested

File: rcute_cozmars/button.py [hunk 2]
```diff
@@ -1,57 +1,56 @@
...
+from. import util
 
 class Button(util.Component):
-    """按钮
+    """ Button
```
Comment (hyansuper) [inline rcute_cozmars/button.py:4]: Do you think it is better without doc string in cases where function names or class names are straight forward？it looks silly to put the same word under def.

**Review by hyansuper — CHANGES_REQUESTED** [review]

**Comment by anjok** [comment]
I force push a commit with the code changes removed, but it would have been ok anyway as python doesn't need the missing spaces. That was the way that translate.google.com returned the text.

As for the "Foo" comments... what I *normally* do at work is not write any docs, expect when it's not totally apparent from the names what the respective thing does. And then mostly abstracts at the class level.

But then again: I'm a pretty experienced developer and work with people like myself. So your target audience may be different. 

If you want more, you can take a look at the Cozmo SDK and see if that provides you with more info than what you would have intuited in the first place.

**Pull request merged**